server/cluster/cluster.go

```diff
@@ -1523,6 +1524,20 @@ func (c *RaftCluster) checkStores() {
 	for _, store := range stores {
 		// the store has already been tombstone
 		if store.IsRemoved() {
+			if store.DownTime() > gcTombstoreInterval {
+				// TODO: remove this lock
```

Comment:
The lock can be ignore , the delete store locked has two step: 1. delete meta from etcd by txn and delete store from store map(have store lock).

Reply:
Because the function name warns me to use the lock.